frontend/src/app/modules/users/users-effects.js

```diff
@@ -1,5 +1,16 @@
 import { client } from '../api/api-client'
 
+
+/**
```

Comment:
why do you have the User object defined twice? I suspect you want to remove one of the definitions....

Reply:
this JSDoc syntax only implies the existence of the User object with such and such properties - I believe it doesn't imply that getUsers() or addUser() return such an object.

also a typo duplicated by copy pasting the code (initilRating)